apps/site/assets/ts/projects/components/FeaturedProjectsRow.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from "react";
+import { SimpleProject as Project } from "./__projects";
+import FeaturedProject from "./FeaturedProject";
+
+interface Props {
+  placeholderImageUrl: string;
+  projectPair: Project[];
+}
+
+const FeaturedProjectsRow = ({
+  projectPair,
+  placeholderImageUrl
+}: Props): ReactElement<HTMLElement> => {
+  const projectToComponent = (project: Project): JSX.Element => (
```

Comment:
also might be better to define `projectToComponent` outside `FeaturedProjectsRow`